clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -41,6 +41,8 @@ import okhttp3.Interceptor
 import okhttp3.OkHttpClient
```

Comment:
Commit message: "When the request interceptor *runs* in a timeout ..."

Reply:
Commit has been removed.